EDMarketConnector.py

```diff
@@ -1171,6 +1174,15 @@ def crewroletext(role: str) -> str:
                     config.set('cmdrs', config.get_list('cmdrs', default=[]) + [monitor.cmdr])
                 self.login()
 
+            if monitor.mode == 'CQC' and entry['event']:
+                err = plug.notify_journal_entry_cqc(monitor.cmdr, monitor.is_beta, entry, monitor.state)
+                if err:
+                    self.status['text'] = err
+                    if not config.get_int('hotkey_mute'):
+                        hotkeymgr.play_bad()
+
+                return  # in CQC
+
             if not entry['event'] or not monitor.mode:
                 # logger.trace('Startup or in CQC, returning')
```

Comment:
If we think we've handled CQC above then this log message, and the return comment, need changing to reflect that this conditional is no longer about that.